src/index.ts

```diff
@@ -1,4 +1,6 @@
 import dotenvSafe = require('dotenv-safe');
+import raven = require('raven');
```

Comment:
This is weird syntax, both line 1 & 2. I suggest to always use module import like this:

```
import dotenvSafe from 'dotenv-safe';
import Raven from 'raven';
```